build_extra/rust/get_cargo_info.py

```diff
@@ -0,0 +1,13 @@
+#!/usr/bin/env python
+# Copyright 2018 the Deno authors. All rights reserved. MIT license.
+
+import os
+import re
+
+# Read the package version from Cargo.toml and output as json
+current_path = os.path.dirname(os.path.realpath(__file__))
+for line in open(os.path.normpath(current_path + "/../../Cargo.toml")):
```

Comment:
`os.path.join(current_path, "../../Cargo.toml")` is better than string concatenation.

Reply:
Thanks for the suggestion, I'll update.